README.md

```diff
@@ -1,7 +1,7 @@
 # Platform Driver Agent
 
-![Passing?](https://github.com/VOLTTRON/volttron-platform-driver/actions/workflows/run_tests.yml/badge.svg)
-[![documentation](https://img.shields.io/badge/docs-mkdocs%20material-blue.svg?style=flat)](https://VOLTTRON.github.io/volttron-platform-driver/)
+![Passing?](https://github.com/eclipse-volttron/volttron-platform-driver/actions/workflows/run_tests.yml/badge.svg)
+[![documentation](https://img.shields.io/badge/docs-mkdocs%20material-blue.svg?style=flat)](https://eclipse-volttron.github.io/volttron-platform-driver/)
```

Comment:
not sure where you got the link to mkdoc, but it doesn't work so remove that.  Unless it won't be made until after this is approved?

Reply:
This link came from the copier template. I'll remove it and also update my copier working branch so we don't autogenerate this link.